### PR TITLE
Hard 119

### PR DESCRIPTION
**Pull request by TotoLulu94**

### CODE REVIEW

File: processing/splitting.py [hunk 1]
```diff
@@ -178,75 +174,409 @@ def create_histogram(o3dcloud, bin_size=0.1, axis =2):
...
+            else :
+                col = math.ceil((x-xmin)/6)
+
+            outfile = rooms['outfile_{}_{}'.format(row, col)]
+            with open(outfile, 'a') as out_file:
+                out_file.write(str(x)+' '+str(y)+' '+str(z)+' '+str(r)+' '+str(g)+' '+str(b)+' '+str(ol)+'\n' )
+                out_file.close()
+    
+    
+print('=========================== 3 =======================================')
```
Comment (nicolas-chaulet) [inline processing/splitting.py]: What is the difference between 1,2 and 3? Could this be encapsulated in a function?

File: providers/datasets.py [hunk 2]
```diff
@@ -47,33 +48,26 @@ def get_info(self,edge_attribs,pc_attribs):
...
             'node_feats': 14 if pc_attribs=='' else len(pc_attribs),
             'edge_feats': edge_feats,
-            'classes': 13,
+            'classes': 14,
             'inv_class_map': {value:key for (key,value) in self.labels.items()},
         }
     
-    def get_datasets(self, args, test_seed_offset=0, single_file = False, filename ='', folder_s = '' ):
-        """ Gets training and test datasets. """
+    def get_data(self, args, filename, folder_s, test_seed_offset=0):
```
Comment (nicolas-chaulet) [inline providers/datasets.py]: If this is for inference purposes only could you please make it explicit in the title of the function?

File: processing/splitting.py (hunk 1 above)
Comment (TotoLulu94) [reply processing/splitting.py]: that was for creating data along a grid with different stride, removing it

File: providers/datasets.py (hunk 2 above)
Comment (TotoLulu94) [reply providers/datasets.py]: done

**Pull request merged**